creator-node/src/services/stateMachineManager/stateReconciliation/updateReplicaSet.jobProcessor.ts

```diff
@@ -60,19 +64,24 @@ module.exports = async function ({
    * on a new replica set. Also, the sync check logic is coupled with a user state on the userStateManager.
    * There will be an explicit clock value check on the newly selected replica set nodes instead.
    */
-  const audiusLibs = await initAudiusLibs(true)
-  const { services: healthyServicesMap } =
-    await audiusLibs.ServiceProvider.autoSelectCreatorNodes({
-      performSyncCheck: false,
-      whitelist: reconfigNodeWhitelist,
-      log: true
-    })
-
-  const healthyNodes = Object.keys(healthyServicesMap || {})
-  if (healthyNodes.length === 0)
-    throw new Error(
-      'Auto-selecting Content Nodes returned an empty list of healthy nodes.'
-    )
+  let audiusLibs = null
+  let healthyNodes = []
+  healthyNodes = await getCachedHealthyNodes()
```

Comment:
only thing to keep an eye out for is there could be like potentially 15 concurrent updates to the redis cache key. this code would solve for that by initializing libs in that case and error if the list is empty. worst case writing 15 to redis should be okay so no-op here. fine here for now but longer term this might best be put into another queue